modules/installation-guide/partials/proc_run-several-workspaces-at-a-time.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// advanced-configuration
+
+[id="run-several-workspaces-at-a-time_{context}"]
+= Running more than one workspace at a time
+
+This procedure describes how to run more than one workspace simultaneously, allowing a user to perform several tasks at once.
```

Comment:
you can run more than one task in a workspace at a time.
the point here is that you can have more than one workspace context running in parallel, per user.
For example you might have a python project that you're working on for work, but a pet project in golang you're doing in parallel while the python compiles.

Sure, you could have both projects in the same workspace, but it might not make sense to do that, and so being able to open more than one at a time is useful.